docs/integrations/sources/microsoft-dynamics-ax.md

```diff
@@ -0,0 +1,10 @@
+# MS Dynamics AX
+
+[MS Dynamics AX](https://dynamics.microsoft.com/en-us/ax) is a powerful enterprise resource planning (ERP) software package for finance and operations.
+
+## Sync overview
+
+MS Dynamics AX runs on the [MSSQL](https://docs.microsoft.com/en-us/dynamics-nav/installation-considerations-for-microsoft-sql-server) database. You can use the [MSSQL connector](mssql.md) to sync your MS Dynamics AX instance by connecting to the underlying database. 
```

Comment:
some of these link to provider docs that don't seem to be the same - link to nav even though the section says AX. double checking this is intended.

Reply:
fixed